modules/fastq_filter.py

```diff
@@ -0,0 +1,77 @@
+def check_gc_content(seq:str, gc_bounds = (0, 100)) -> bool:
+    '''
+    Compute GC-content of the DNA seqence and check if it fits into bounds.
+
+    Arguments:
+    - seq (str): DNA sequence
+    - gc_bounds (tuple or int, default = (0, 100)): GC-content bounds in percent.
+    Tuple if contains lower and upper bounds, int if only 
+    contains an upper bound.
+
+    Returns: boolean
+    '''
+    gc_count= 0
+    for nucleotide in seq:
+        if nucleotide == 'G' or nucleotide == 'C':
+            gc_count += 1
+    gc_content = (gc_count /len(seq)) * 100
+
+    if isinstance(gc_bounds, tuple):
+            lower_bound = gc_bounds[0]
+            upper_bound = gc_bounds[1]
+            if gc_content >= lower_bound and gc_content <= upper_bound:
+                return True
+            else:
+                return False
+    else:
+        if gc_content < gc_bounds:
+            return True
+        else:
+            return False
+
+
+def check_length(seq:str, length_bounds = (0, 2**32))-> bool:
+    '''
+    Check if length of the input DNA seqence fits into bounds.
+
+    Arguments:
+    - seq (str): DNA sequence
+    - length_bounds (tuple or int, default = (0, 2**32)): corresponds to its name. 
+    Tuple if contains lower and upper bounds, int if only
+    contains an upper bound.
+
+    Returns: boolean
+    '''
+    if isinstance(length_bounds, tuple):
+            lower_bound = length_bounds[0]
+            upper_bound = length_bounds[1]
+            if len(seq) >= lower_bound and len(seq) <= upper_bound:
+                return True
+            else:
+                return False
+    else:
+        if len(seq) < length_bounds:
+            return True
+        else:
+            return False
```

Comment:
Тут сразу несколько проблемок. Не по порядку:
1) На надо делать `if return True else return False`, просто возвращаем результат булевой операции
2) Получается в зависимости от типа аргумента функция разбита на 2 ветви логики. Получается очень громоздко. Лучше было бы свети один случай к другому и написать код общего кода. 
3) Ну и для красоты можно было бы улсовие на принадлежность интервалу записать чуть по другому

Например, так:

```suggestion
    if isinstance(length_bounds, int) or  isinstance(length_bounds, float):
        length_bounds = (0, length_bounds)

    return length_bounds[0] <= len(seq) <= length_bounds[1]
```